df12_pages/config/loader.py

```
@@ -218,4 +241,158 @@ def _build_page_config(
+            default_theme=default_theme,
+            fallback_footer=fallback_footer,
+        )
+    return result
+
+
+def _build_subsite_config(  # noqa: PLR0913
+    *,
+    key: str,
+    payload: typ.Mapping[str, typ.Any],
+    page_defaults: _PageDefaults,
+    shared_layouts: typ.Mapping[str, typ.Any],
```

Ruff (0.15.7)

[warning] Unused function argument: `shared_layouts`

(ARG001)